tools/accuracy_checker/README.md

```diff
@@ -68,13 +68,15 @@ You can use any of them or several at a time. For correct work, Accuracy Checker
 If all prerequisite are installed, then you are ready to install **accuracy checker**:
 
 ```bash
+pip install -r requirements.in
```

Comment:
added this since setup.py doesn't realize if there is pre-release package version or not and always downloads the newest one

> easy_install has never had support for excluding pre-releases other than by excluding a range that includes pre-releases

This happened to scikit-learn package, v1.0rc1 were uploaded as pre-release 7.09.2021, setup.py started to download v1.0rc1 version instead of latest release 0.24.2

Reply:
do not agree with this!!! I give approve only on requirements.in fix

Reply:
we have diiferent installation modes, it is not true for all and besides that not install all dependencies for default mode

Reply:
ok, removed this